css-ui-4/Overview.bs

```diff
@@ -287,13 +287,14 @@ Animation type: by computed value
 
 <pre class="propdef">
 Name: outline-color
-Value: <<color>> | invert
+Value: [ <<color>> || <<1d-image>> ] | invert
 Initial: invert
 Applies to: all elements
 Inherited: no
 Percentages: N/A
 Computed value: The computed value for ''outline-color/invert'' is ''outline-color/invert''.
  For <<color>> values, see [[!CSS-COLOR-4#resolving-color-values]] in [[!CSS-COLOR-4]].
+ <<1d-image>> values are computed as defined in their definition.
```

Comment:
I suggest using the same sentence construction as the `<<color>>` type, for consistency.

Reply:
I've done that. @fantasai Please have a look whether that's what you had in mind!

Sebastian

Reply:
I've fixed the two links now. The one to Color 4 was actually broken.